starknet_py/net/account/account_client.py

```diff
@@ -434,6 +435,47 @@ async def sign_declare_transaction(
 
         return dataclasses.replace(declare_tx, signature=signature, max_fee=max_fee)
 
+    async def sign_deploy_account_transaction(
+        self,
+        *,
+        class_hash: int,
+        contract_address_salt: int,
+        constructor_calldata: Optional[List[int]] = None,
+        max_fee: int,
+    ) -> DeployAccount:
+        """
+        Create and sign deploy account transaction
+
+        :param class_hash: Class hash of the contract class to be deployed
+        :param contract_address_salt: A salt used to calculate deployed contract address
+        :param constructor_calldata: Calldata to be passed to contract constructor
+            and used to calculate deployed contract address
+        :param max_fee: Max fee to be paid for deploying account transaction. Enough tokens must be prefunded before
+            sending the transaction for it to succeed.
+        :return: Signed DeployAccount transaction
+        """
+        # TODO not sure if this should be added, but the example transaction from starkware uses version 1
+        if self.supported_tx_version != 1:
+            raise ValueError(
+                "Signing deploy account transactions is only supported with transaction version 1"
+            )
```

Comment:
I've checked that and this should be added
![image](https://user-images.githubusercontent.com/61014013/194545495-e6d922e9-be25-41ee-a838-8dc1234def85.png)